types/pacote/index.d.ts

```diff
@@ -66,7 +66,7 @@ export interface CommonMetadata {
     homepage?: string | undefined;
     keywords?: string[] | undefined;
     license?: string | undefined;
-    maintainers?: Person[] | undefined;
+    maintainers: Person[];
```

Comment:
Why did this become non-optional?

Reply:
Maybe this is an unrelated change (fix?)?  It doesn't seem to appear in either Abbreviated interface or in the new tests.

Reply:
Maybe this is an unrelated change (fix?)?  It doesn't seem to appear in either Abbreviated interface or in the new tests.